EIPS/eip-2733.md

```diff
@@ -3,7 +3,7 @@ eip: 2733
 title: Transaction Package
 author: Matt Garnett (@lightclient)
 discussions-to: https://ethereum-magicians.org/t/eip-transaction-package/4365
-status: Stagnant
+status: Withdrawn
```

Comment:
Should have `withdrawal-reason:`.

Reply:
Hmm, shouldn't CI have caught that?  Or did we just make it so CI won't error on having a withdrawal reason, but we didn't add logic to make it fail on missing withdrawal reason?

Reply:
Ain't nobody got time to implement complex checks in eipv 😅